storage/src/block_meta_storage.rs

```diff
@@ -27,14 +28,16 @@ pub trait BlockMetaStorageReader: Sync + Send {
     fn get_live_blocks(&self, block_hash: BlockHash, max_ttl: usize) -> Result<Vec<BlockHash>, StorageError>;
 }
 
-#[derive(Clone)]
+#[derive(Clone, Getters)]
 pub struct BlockMetaStorage {
-    kv: Arc<BlockMetaStorageKV>
+    kv: Arc<BlockMetaStorageKV>,
+    #[get = "pub"]
```

Comment:
@adonagy 
I would hide access to predecessors_inde here,
and add `fn store_predecessors` to BlockMetaStorage, which just calls inside `predecessors_index.store_predecessors(..)`